openmetadata-ui/src/main/resources/ui/src/components/TestSuitePipelineTab/TestSuitePipelineTab.component.tsx

```diff
@@ -490,30 +494,21 @@ const TestSuitePipelineTab = () => {
                   <Button
                     data-testid="logs"
                     disabled={!viewPermission}
+                    href={getLogsViewerPath(
+                      PipelineType.TestSuite,
+                      record.service?.name || '',
+                      record.fullyQualifiedName || ''
+                    )}
+                    target="_blank"
```

Comment:
Do we need to open it in new tab?

Reply:
@chirag-madlani , No, we can open the page in the same tab, I will remove that.